examples/workflow/pygrb/postprocessing.ini

```diff
@@ -0,0 +1,97 @@
+[workflow-postproc]
```

Comment:
Looks good to me.  Just one thing: we should try combining common options in [efficiency] and [inj_efficiency] using [efficiency&inj_efficiency], so we don't have to track the same options in two different places.  I know there were issues with the use of & yesterday, however.